### ISSUE TITLE
There are 3 bytes that needs to be changed in the savefile after .wc6 injector

### ISSUE THREAD

**Issue by BernardoGiordano** [BernardoGiordano/EventAssistant#2]
After some researches, i found that, after succesfully injecting a .wc6 into an OR/AS' main, i need to change other 3 bytes, apparently in a way i can not figure out.

#1 : there's a byte that locates around 117700 and 118000 (apparently in a casual way) that is linked in some ways to the wondercard i want to inject

#2 #3 : bytes located at 483170 and 483171. I don't know what determines the change of those bytes.

If anyone has idea of how to work on it, he can explain to me or make a pull request continuing the code that already is in the repo.

**Comment by kwsch** [comment]
1: The 'received' flag for the wonder card. It's so that the game can prevent you from receiving the same card after you've discarded it. There's 0x100 bytes (8*256 bitflags).

2: Block checksum. Save files have integrity checks to check for corrupted data.

https://projectpokemon.org/wiki/Pok%C3%A9mon_ORAS_Save_File_Structure
https://github.com/gocario/PHBank/blob/a956ab39060148c8dafdb1f472c96632b22dcc65/source/save_manager.cpp#L1255-L1367

Basically you need to calculate the ccitt-16 checksum over the block (0x22000-0x23A90 for ORAS) and update the block checksum at the end of the file.

**Comment by BernardoGiordano** [comment]
hello! Thank you for the answer, this will save me lots of time. If i understand well, i have to modify the rewriteSaveCHK function to have a blockcount of 2 and a csoff of 483170, to update the block checksum at the eof or those values are fixed? If this is correct, the same procedure can be made with the 0x1CC00-0x1E690 block
